fractals/sierpinski_triangle.py

```diff
@@ -1,76 +1,85 @@
-#!/usr/bin/python
-
-"""Author Anurag Kumar | anuragkumarak95@gmail.com | git/anuragkumarak95
-
-Simple example of Fractal generation using recursive function.
-
-What is Sierpinski Triangle?
->>The Sierpinski triangle (also with the original orthography Sierpinski), also called
-the Sierpinski gasket or the Sierpinski Sieve, is a fractal and attractive fixed set
-with the overall shape of an equilateral triangle, subdivided recursively into smaller
-equilateral triangles. Originally constructed as a curve, this is one of the basic
-examples of self-similar sets, i.e., it is a mathematically generated pattern that can
-be reproducible at any magnification or reduction. It is named after the Polish
-mathematician Wacław Sierpinski, but appeared as a decorative pattern many centuries
-prior to the work of Sierpinski.
+"""
+Author Anurag Kumar | anuragkumarak95@gmail.com | git/anuragkumarak95
 
-Requirements(pip):
-  - turtle
+Simple example of fractal generation using recursion.
 
-Python:
-  - 2.6
+What is the Sierpiński Triangle?
+    The Sierpiński triangle (sometimes spelled Sierpinski), also called the
+Sierpiński gasket or Sierpiński sieve, is a fractal attractive fixed set with
+the overall shape of an equilateral triangle, subdivided recursively into
+smaller equilateral triangles. Originally constructed as a curve, this is one of
+the basic examples of self-similar sets—that is, it is a mathematically
+generated pattern that is reproducible at any magnification or reduction. It is
+named after the Polish mathematician Wacław Sierpiński, but appeared as a
+decorative pattern many centuries before the work of Sierpiński.
 
-Usage:
-  - $python sierpinski_triangle.py <int:depth_for_fractal>
+Requirements (pip): turtle
```

Comment:
`turtle` builtin, not installed via pip... https://docs.python.org/3/library/turtle.html
```suggestion
```